docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM php:8.2-apache
```

Comment:
I would recommend using multi-stage build to avoid building all of this each time and why not nginx?

Reply:
I don't see any advantages of using nginx on single host for development purposes (this docker image is not intended for production).

About multi-stage build: thanks, didn't knew about that one. Will dive into it, and will see what can be done.